carbon-design-system/carbon-for-ibm-dotcom
# fix(card-inverse): use $inverse-focus-ui token for the Card Inverse focus style

### Related Ticket(s)

DDS Consulting Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6594
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2236

### Description
- This PR uses the `$inverse-focus-ui` token for the Card Inverse focus style.

### Testing Instructions

Focus inverse on White
![Screen Shot 2021-10-06 at 12 52 09 PM](https://user-images.githubusercontent.com/1815714/136248856-1fc1a97a-1272-420a-932a-e8220723f0e5.png)

Focus inverse on G100
![Screen Shot 2021-10-06 at 12 52 00 PM](https://user-images.githubusercontent.com/1815714/136248890-e30fbfa1-71cd-4515-9fed-7c02026b9540.png)

- Visit the `Feature Section` layout Component - `/?path=/story/components-feature-section--default`.
- Tab focus on the card
- Ensure that the focus line is pulling hexcode color from the `$inverse-focus-ui` token.
